app/views/partials/_header.html.erb

```diff
@@ -5,10 +5,10 @@
   </div>
 
   <ul class="menu">
-    <li role="presentation" <%= users_active? %>>
+    <li role="presentation" class=<%= users_active_class %>>
       <%= link_to 'Team', leaderboard_users_path %>
     </li>
-    <li role="presentation" <%= projects_active? %>>
+    <li role="presentation" class=<%= projects_active_class %>>
```

Comment:
@mauro-oto one last thing, could you add the html quotes here? it currently ends up like ` class=active>`